src/engine/api-factory.ts

```diff
@@ -18,6 +19,7 @@ export abstract class ApiFactory<T extends Api = any> {
       events: this.events,
       name: this.profile.name,
       profile: this.profile,
+      render: this.render ? () => this.render() : undefined,
```

Comment:
i m not quite sure about the syntax here... is the `return` keyword  missing here?

Reply:
`render: this.render ? () => this.render() : undefined,` is the same as : 
```javascript
render: () => {
  if (!!this.render) {
    return () => { return this.render() }
  } else {
    return undefined
  }
}
```